diff -crB wpa-2.3/src/radius/radius_client.c wpa-2.3_patch/src/radius/radius_client.c
*** wpa-2.3/src/radius/radius_client.c	Thu Oct  9 14:41:31 2014
--- wpa-2.3_patch/src/radius/radius_client.c	Mon Jan 12 01:17:14 2015
***************
*** 658,664 ****
  	}
  
  	if (msg_type == RADIUS_ACCT || msg_type == RADIUS_ACCT_INTERIM) {
! 		if (conf->acct_server == NULL || radius->acct_sock < 0) {
  			hostapd_logger(radius->ctx, NULL,
  				       HOSTAPD_MODULE_RADIUS,
  				       HOSTAPD_LEVEL_INFO,
--- 658,664 ----
  	}
  
  	if (msg_type == RADIUS_ACCT || msg_type == RADIUS_ACCT_INTERIM) {
! 		if (conf->acct_server == NULL) {
  			hostapd_logger(radius->ctx, NULL,
  				       HOSTAPD_MODULE_RADIUS,
  				       HOSTAPD_LEVEL_INFO,
***************
*** 672,678 ****
  		s = radius->acct_sock;
  		conf->acct_server->requests++;
  	} else {
! 		if (conf->auth_server == NULL || radius->auth_sock < 0) {
  			hostapd_logger(radius->ctx, NULL,
  				       HOSTAPD_MODULE_RADIUS,
  				       HOSTAPD_LEVEL_INFO,
--- 672,678 ----
  		s = radius->acct_sock;
  		conf->acct_server->requests++;
  	} else {
! 		if (conf->auth_server == NULL) {
  			hostapd_logger(radius->ctx, NULL,
  				       HOSTAPD_MODULE_RADIUS,
  				       HOSTAPD_LEVEL_INFO,
